js/chatbot/src/ChatBot.svelte

```diff
@@ -91,6 +95,16 @@
 			node.style.position = "relative";
 			node.appendChild(copy_div);
 		});
+
+		renderMathInElement(document.body, {
+			delimiters: [
+				{ left: "$$", right: "$$", display: true },
+				{ left: "$", right: "$", display: false },
+				{ left: "\\(", right: "\\)", display: false },
+				{ left: "\\[", right: "\\]", display: true }
```

Comment:
I think we should remove these two ways of defining LaTeX to avoid ambiguity. In practice, the $ sign approach is the most common way to define LaTeX and we should just stick to that
```suggestion
```

Reply:
Sounds good. Will probably prevent sneaky bugs/ syntax collisions down the road. We just need to document it.